.npmignore

```diff
@@ -1,4 +1,2 @@
-misc/
 node_modules/
-src/
```

Comment:
We can remove these because the package.json "files" field is a whitelist; but we do have to have an `.npmignore` file or npm will use the `.gitignore`, which ignores some things we want in the published package.